go-runtime/ftl/reflection.go

```diff
@@ -0,0 +1,24 @@
+package ftl
+
+import (
+	"runtime"
+	"strings"
+)
+
+// Module returns the FTL module currently being executed.
+func Module() string {
```

Comment:
👏 I was wanting this more generaly and didnt want to call `config.go`'s `callerModule()`
And it seems that pfi wanted it in some cases as well: https://github.com/tbdeng/pfi/blob/ef7ee116ca4dc269dc9cc857187e2b0f5e697489/backend/modules/tbdex/offerings/offerings.go#L17